pkg/services/baremetal/remediation/remediation.go

```diff
@@ -55,7 +55,7 @@ func (s *Service) Reconcile(ctx context.Context) (res reconcile.Result, err erro
 	host, err := s.getHost(ctx)
 	if err != nil {
 		if apierrors.IsNotFound(err) {
-			err = fmt.Errorf("HetzerBareMetalMachine not found")
+			err = fmt.Errorf("HetznerBareMetalHost not found not found")
```

Comment:
I think there is not found twice :sweat_smile:

Reply:
huh 😓